src/openforms/utils/fields.py

```diff
@@ -4,6 +4,12 @@
 
 
 class StringUUIDField(models.UUIDField):
+    """
+    DEPRECATED
+
+    remains here for references in historical migrations
+    """
+
```

Comment:
https://docs.djangoproject.com/en/2.2/topics/migrations/#considerations-when-removing-model-fields this is a better option to actually remove the field :)

Reply:
Indeed, I'll replace the comment.